main/rust/template.py

```diff
@@ -76,6 +76,13 @@ def post_patch(self):
     cargo.clear_vendor_checksums(self, "libc-0.2.155")
 
 
+def init_configure(self):
```

Comment:
doesn't need to be a hook

Reply:
mfs be like 'doesnt need to be x' without saying 'do y'

Reply:
tool_flags=

Reply:
just put it in toplevel tool_flags as i said on irc